src/utils/ViewCount.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+function ViewCount({viewCount}) {
+
+    
+  return (
+      <span>{Number(viewCount) > 1000 ?
+          `${Math.floor(Number(viewCount) / 1000)}K`
+          : Number(viewCount) > 1000000
+              ? `${Math.floor(Number(viewCount) / 1000000)}M`
+              : `${Number(viewCount)}`} views
+      </span>
```

Comment:
Check the implementation, sort properly again.

Reply:
okay update the condition